class.jetpack.php

```diff
@@ -584,6 +584,9 @@ private function __construct() {
 			Jetpack_Search_Performance_Logger::init();
 		}
 
+		// detect amp-wp and modify output as appropriate
+		require_once JETPACK__PLUGIN_DIR . '3rd-party/class.jetpack-amp-support.php';
```

Comment:
If we're just running this in the construct without a specific condition, should it go in the main [3rd-party](https://github.com/Automattic/jetpack/blob/master/3rd-party/3rd-party.php) file instead? That file should be loaded before the `devicepx` method in the `Jetpack` class is called.

Reply:
Good call, fixed.